src/libcore/mem/maybe_uninit.rs

```diff
@@ -151,7 +153,7 @@ use crate::mem::ManuallyDrop;
 /// let mut data_len: usize = 0;
 ///
 /// for elem in &mut data[0..500] {
-///     unsafe { ptr::write(elem.as_mut_ptr(), String::from("hello")); }
+///     *elem.as_mut_ptr() = MaybeUninit::new(String::from("hello"));
```

Comment:
Should be `*elem = MaybeUninit::new(String::from("hello"));`

Reply:
Yeah, I mistakenly left the `.as_mut_ptr()`.